ArrayDemo.cls

```diff
@@ -0,0 +1,10 @@
+public class ArrayDemo {
```

Comment:
@pranalimm 
This is wrong implementation.
> Method which will accept integer parameter and build an print array of upto that integer
parameter.
(For example : If your accepts (integer) 5 as parameter then build array of any
datatype but array size should be accepted parameter so in this case your array size
should be 5 )

Here you are creating list of any size. Array is not with fix size